terraform/domains/environment_domains/config/production.tfvars.json

```diff
@@ -0,0 +1,23 @@
+{
+  "hosted_zone": {
+    "schoolexperience.education.gov.uk": {
+      "front_door_name": "s189p01-gsedomains-fd",
+      "resource_group_name": "s189p01-gsedomains-rg",
+      "domains": [
+        "apex"
+      ],
+      "cached_paths": [
+        "/packs/*"
+      ],
+      "environment_short": "pd",
+      "origin_hostname": "school-experience-app-production.london.cloudapps.digital",
+      "null_host_header": true,
+      "cnames": {
```

Comment:
Are we not migrating all the existing cnames?

Reply:
There are no other cnames which are subdomains of schoolexperience.education.gov.uk. Staging uses staging-schoolexperience.education.gov.uk.